DQM/SiPixelMonitorTrack/src/SiPixelTrackResidualSource.cc

```diff
@@ -976,18 +964,30 @@ void SiPixelTrackResidualSource::analyze(const edm::Event& iEvent, const edm::Ev
   const edm::DetSetVector<PixelDigi> diginp = *(digiinput.product());
   
 
-  if(debug_){
-    std::cout << "Trajectories\t : " << trajColl.size() << std::endl;
-    std::cout << "recoTracks  \t : " << trackColl.size() << std::endl;
-    std::cout << "Map entries \t : " << ttac.size() << std::endl;
-  }
-  
   std::set<SiPixelCluster> clusterSet;
   TrajectoryStateCombiner tsoscomb;
   int tracks=0, pixeltracks=0, bpixtracks=0, fpixtracks=0; 
   int trackclusters=0, barreltrackclusters=0, endcaptrackclusters=0;
   int otherclusters=0, barrelotherclusters=0, endcapotherclusters=0;
 
+  //get trajectories
+  edm::Handle<std::vector<Trajectory> > trajCollectionHandle;
+  iEvent.getByToken ( tracksrcToken_, trajCollectionHandle );
+  if (trajCollectionHandle.isValid()) {
+
+  auto const & trajColl = *(trajCollectionHandle.product());
+  //get the map
+  edm::Handle<TrajTrackAssociationCollection> match;
+  iEvent.getByToken( trackAssociationToken_, match);
+  auto const &  ttac = *(match.product());
```

Comment:
I think this file is "obsolete" as its functionality will be replaced by the new PhaseI code